<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomMenuButton</class>
 <widget class="QWidget" name="CustomMenuButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBoxCustomMenuButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblCustomMenuButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </widget>
  <widget class="scopy::gui::CustomPushButton" name="btnCustomMenuButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::CustomPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/custom_push_button.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
